selfdrive/car/hyundai/values.py

```diff
@@ -43,10 +43,13 @@ class Buttons:
 
 CAMERA_MSGS = [832, 1156, 1191, 1342]
 
-CHECKSUM = {
-  "crc8": [CAR.SANTA_FE],
-  "6B": [CAR.KIA_SORENTO, CAR.GENESIS],
-  "7B": [CAR.KIA_STINGER, CAR.ELANTRA],
+# Lane Keep Assist related Features and Limitations
+LKAS_FEATURES = {
+  "6B": [CAR.KIA_SORENTO, CAR.GENESIS],     # 6 Bytes used in Checksum
+  "7B": [CAR.KIA_STINGER, CAR.ELANTRA],			# 7 Bytes used in Checksum
+  "crc8": [CAR.SANTA_FE],		                # CRC Checksum
+  "icon_basic": [CAR.GENESIS],		          # Anything but 2 for LKAS_Icon causes MDPS Fault
```

Comment:
It seems unused. Why not keeping this CHECKSUM specific?

Reply:
This is used, hyundaican line 11.
As for the soft disable, this usage may need to be in its own PR as it may be .... rejected.
I'll remove it for this PR

Reply:
Can keep it checksum specific and make one for vehicle limitations seperate if preferred?
I am wishing to remove any vehicle specific code out of everything except values and interface.

Reply:
yeah, icon_basic has nothing to do with the algorithm used for CHECKSUM computation. I would prefer to use 2 different params.